### PR TITLE
SISRP-4519 - Changed broken link in _widgets.scss file

### PR DESCRIPTION
**Pull request by davidlimcheng**
https://jira.berkeley.edu/browse/SISRP-4519

### CODE REVIEW

File: src/assets/stylesheets/_widgets.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@
   background: $cc-color-white;
   margin: 15px 15px 0 0;
 
-  // See https://github.com/h5bp/html5-boilerplate/blob/master/css/main.css
+  // See http://git.io/vITBH
```
Comment (christianv) [inline src/assets/stylesheets/_widgets.scss]: @davidlimcheng can we change this to the `https` version?

**Comment by christianv** [comment]
@davidlimcheng also would be good if you can include the ticket number in the commit message:

`SISRP-4519 - Changed broken link in _widgets.scss file`

**Comment by davidlimcheng** [comment]
Changed from http to https - will start adding ticket number to commit messages.  Hope this is better!

**Comment by christianv** [comment]
@davidlimcheng :+1:

**Pull request merged**